conjurapi/logging/logging.go

```diff
@@ -0,0 +1,35 @@
+package logging
+
+import (
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+var ApiLog = logrus.New()
+
+func init() {
+	dest, ok := os.LookupEnv("CONJURAPI_LOG")
+	if !ok {
+		return
+	}
+
+	var out io.Writer
+	switch dest {
```

Comment:
Would be nice to do a straight printout to stdout about what will be the new logging target for easier debugging.

Reply:
To stderr, maybe? I can add that.